hw/bsp/dialog_da1469x-dk-pro/otp_tool.py

```diff
@@ -593,16 +616,18 @@ def test_alive_target(uart):
         raise SystemExit("Failed to write to %s" % uart)
 
     data = ser.read(rlen)
-    print(data)
     if len(data) != rlen:
         raise SystemExit("Failed to receive response, exiting")
 
     response = cmd_response._make(struct.unpack_from('IIII', data))
+    if response.som != 0x55aa55aa:
+        raise SystemExit("SOM not valid, invalid response")
```

Comment:
Some copy/paste here. Although the checks must be done there could be a bit or reuse so in case `0x55aa55aa` or the error message needs to change no find/replace is required.